internal/pyxis/pyxis.go

```diff
@@ -408,12 +408,23 @@ func (p *pyxisClient) updateProject(ctx context.Context, certProject *CertProjec
 	return &newCertProject, nil
 }
 
-func (p *pyxisClient) createTestResults(ctx context.Context, testResults *TestResults) (*TestResults, error) {
+func (p *pyxisClient) createOrUpdateTestResults(ctx context.Context, testResults *TestResults, httpMethod string) (*TestResults, error) {
```

Comment:
This seems... clunky. Why not have a createTestResults, and an updateTestResults? httpMethod seems to me to expose far to much about the implementation.

Reply:
I've switch to adding a new `updateTestResults` method, which should resolve this.